geniesp/bpc_redcap_export_mapping.py

```diff
@@ -245,21 +245,21 @@ def get_drug_mapping(
     var_names = []
 
     # ! Not removing data dictionary code until 100% certain
-    # synid_file_dd = _get_synid_dd(syn, cohort, synid_table_prissmm)
+    synid_file_dd = _get_synid_dd(syn, cohort, synid_table_prissmm)
 
-    # dd = pd.read_csv(
-    #     syn.get(synid_file_dd).path, encoding="unicode_escape", low_memory=False
-    # )
-    grs = pd.read_csv(
-        syn.get(synid_file_grs).path, encoding="unicode_escape", low_memory=False
```

Comment:
I'm thinking that you might be able to remove`synid_file_grs` from the parameter of this function as well.

Reply:
Thanks Dan, there's some history to this as we have continuously gone back and forth from removing, then adding back in, then removing again. That's why I'm leaving in the overall structure for now until we are absolutely certain GRS isn't going to be used